### PR TITLE
Fixing warnings

### PR DESCRIPTION
**Pull request by peter-kehl**
Dear Selenium IDE team,

Good that we are resurrecting this for our users.

I've fixed several hidden warnings. They showed up in Firefox Browser Console. Some were incompatible with Strict Javascript.

Looking forward to working together.

### CODE REVIEW

File: src/atoms.js [hunk 1]
```diff
@@ -10594,7 +10594,7 @@ core.events.controlKeyDown_ = !1;
 core.events.altKeyDown_ = !1;
 core.events.metaKeyDown_ = !1;
 core.events.shiftKeyDown_ = !1;
-var XPCNativeWrapper = XPCNativeWrapper || function(a) {};
```
Comment (lukeis) [inline src/atoms.js]: this file is auto generated from the selenium project. you'll want to fix this here instead:

https://github.com/SeleniumHQ/selenium/blob/master/javascript/selenium-atoms/events.js#L47

Comment (corevo) [reply src/atoms.js]: Correct, I'd like to refrain changing atoms, I'm working on compiling them from source and will delete that file soon enough.

**Comment by peter-kehl** [comment]
Removed the change to atoms.js. Please merge.

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ We are using [SideeX](http://sideex.org/) as a start point. The SideeX team was
 ```peru sync```  
 ```yarn```
 - Build the extension  
-```yarn build```
+```yarn build``` (that runs `build` command from [`package.json`](./package.json). See documentation of [yarn run](https://yarnpkg.com/en/docs/cli/run) and [package.json > scripts](https://yarnpkg.com/en/docs/package-json#toc-scripts).)
```
Comment (corevo) [inline README.md]: Would you mind suggesting this change in a different PR?

**Comment by peter-kehl** [comment]
Removed changes to README.md. Please merge.

**Comment by corevo** [comment]
If you want to submit README changes please submit them in a different PR

**Pull request merged**